custom_components/powercalc/data/signify/1742930P7/model.json

```
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Signify, 1742930P7",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 1,
+        "SLEEP_TIME": 2,
+        "VERSION": "v0.18.4:docker"
+    },
+    "name": "Hue Outdoor Wall",
```

Let's call this "Hue Impress Outdoor Wall Light", as it's the Impress outdoor light, and people will search on that. This name is displayed in supported models list.

---

```suggestion
    "name": "Hue Impress Outdoor Wall Light Small",
```

I've added small because there is also a large variant